docs/sources/tutorials/build-a-streaming-data-source-plugin/index.md

```diff
@@ -45,7 +45,7 @@ To build a backend for your data source plugin, Grafana requires a binary that i
 The easiest way to get started is to use the Grafana [create-plugin tool](https://www.npmjs.com/package/@grafana/create-plugin). Navigate to the plugin folder that you configured in step 1 and type:
```

Comment:
change "type" to "enter" (refer to https://developers.google.com/style/word-list)